alvr/client_openxr/src/lib.rs

```diff
@@ -603,27 +622,14 @@ pub fn entry_point() {
                     }
                 }
 
-                views = history_views.unwrap_or_else(|| {
-                    let default_view = xr::View {
-                        pose: xr::Posef {
-                            orientation: xr::Quaternionf {
-                                x: 0.0,
-                                y: 0.0,
-                                z: 0.0,
-                                w: 1.0,
-                            },
-                            position: xr::Vector3f::default(),
-                        },
-                        fov: xr::Fovf {
-                            angle_left: -0.1,
-                            angle_right: 0.1,
-                            angle_up: 0.1,
-                            angle_down: -0.1,
-                        },
-                    };
-
-                    vec![default_view, default_view]
-                });
+                views = if hardware_buffer.is_null() || history_views.is_none() {
+                    vec![last_good_views[0], last_good_views[1]]
+                } else {
+                    let unwrapped_views = history_views.unwrap();
+                    last_good_views[0] = unwrapped_views[0];
+                    last_good_views[1] = unwrapped_views[1];
+                    unwrapped_views
+                };
```

Comment:
I would rewrite this as:
```suggestion
                views = if let Some(views) = history_views {
                    last_good_views = views.clone();
                    views
                } else {
                    last_good_views.clone();
                };
```
Avoid using `.unwrap()` like a plague :)